requirements.txt

```diff
@@ -1,6 +1,7 @@
 click==6.2
 PTable==0.9.2
 netaddr==0.7.18
+nsot==1.1.1
```

Comment:
Yeah I feel like it's ok to dupe `slugify()` for now since it's so simple. We can revisit it if it becomes an issue. But I'm worried about tightly coupling the client and server code for the near term.